File: Examples/Ruby/Cells/post_set_cell_html_string.rb
require 'aspose_cells_cloud'

class Cell

  include AsposeCellsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("da77c9f5da014d91faf2961ecec2de66", "B01A15E5-1B83-4B9A-8EB3-0F2BFA6AC766")
    @cells_api = CellsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_set_cell_html_string
    file_name = "myWorkbook.xlsx"
    upload_file(file_name)

    sheet_name = "Sheet1"
    cell_name = "A3"
    response = @cells_api.post_set_cell_html_string(file_name, sheet_name, cell_name)
  end

end

cell = Cell.new()
puts cell.post_set_cell_html_string

File: Examples/PHP/Cells/PostSetCellHtmlString.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Cells\CellsApi;
use Aspose\Cells\AsposeApp;

class Cell {

	public $cells;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->cells = new CellsApi();
	}

	public function postSetCellHtmlString() {
		// Upload file to Aspose Cloud Storage
		$fileName = "myWorkbook.xlsx";
		Utils::uploadFile($fileName);

		//$file = realpath(__DIR__ . '/../../..') . '/Data/index.html';
        $result = $this->cells->PostSetCellHtmlString($name=$fileName, $sheetName="Sheet1", $cellName="A3", $storage = null, $folder = null, $file = null);
        print_r($result);
	}
}

$cell = new Cell();
$cell->postSetCellHtmlString();

?>